src/dppm_rest_api/actions/app.cr

```diff
@@ -1,4 +1,7 @@
 require "../../utils"
+require "dppm"
+require "../../config/route"
+require "../../utils"
```

Comment:
You shouldn't backward require modules, that the purpose of organizing our code in trees.
This simplify testing and refactoring, especially when changing paths